### PR TITLE
Feature x118 Swagger and Redoc documentations have been added.

### PR DESCRIPTION
**Pull request by MykhailoGrynenko**

### CODE REVIEW

**Review by kushnir-tadey — APPROVED** [review]

**Comment by v-lavrushko** [comment]
but where?

**Comment by MykhailoGrynenko** [comment]
> but where?

/swagger or /redoc and you will see.

**Review by v-lavrushko — APPROVED** [review]

**Review by Maxgioman — APPROVED** [review]
so many new packages. do we really need them all?

File: ProjectShop/requirements.txt [hunk 1]
```diff
@@ -3,20 +3,18 @@ certifi==2021.5.30
...
-six==1.16.0
-sqlparse==0.4.1
 djangorestframework-simplejwt==4.8.0
+drf-yasg==1.20.0
 idna==3.2
 phonenumbers==8.12.31
 Pillow==8.3.2
 PyJWT==2.1.0
 pytz==2021.1
+six==1.16.0
```
Comment (strangerok1507) [inline ProjectShop/requirements.txt:17]: я бы это тоже удалил

Comment (MykhailoGrynenko) [reply ProjectShop/requirements.txt]: Done

**Review by strangerok1507 — APPROVED** [review]

**Comment by MykhailoGrynenko** [comment]
closes #119

**Pull request merged**